app/frontend_messages.py

```diff
@@ -0,0 +1,4 @@
+messages = {
+  'MULTIPLE_SURVEYS': "Unfortunately you can only complete one survey at a time.",
```

Comment:
You may want to consider more descriptive/long constant names if this dictionary is going to grow